tests/19-get-spanning-data.rs

```diff
@@ -0,0 +1,30 @@
+use modular_bitfield::prelude::*;
+use std::convert::TryFrom;
+
+#[bitfield]
+#[derive(Copy, Clone, Eq, PartialEq)]
+pub struct ColorEntry {    
+    r : B5,    
+    g : B5,
+    b : B5,
+    unused: B1,
+}
+
+fn main() {
+    for i in 0..=std::u16::MAX {
```

Comment:
this test, albeit being extensive will probably take way too long to execute for a unit test. not tested though. however, unit tests should always be implemented in a way that they take up a minimum amount of time. iterating through all `u16` values to me seems to be a bit too much but maybe i am wrong and highly underestimate the power of modern CPUs.